net/qpid-proton/Portfile

```diff
@@ -19,13 +19,14 @@ homepage            https://qpid.apache.org
 
 PortGroup           github 1.0
 
-github.setup        apache qpid-proton 0.26.0
+github.setup        apache qpid-proton 0.27.1
 
 PortGroup           cmake 1.0
```

Comment:
Please switch to the cmake 1.1 portgroup. See https://trac.macports.org/ticket/58426. 

This means that you can then remove the line:

`cmake.out_of_source yes
`

Reply:
@mf2k Thanks for the review and will do

Reply:
@mf2k Made and tested the change locally with no problems. Let me know if there is anything further I need to address.